ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -31,12 +31,17 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.10.2') {
+
+    implementation ('com.github.vungle:vungle-android-sdk:6.10.3-RC3') {
```

Comment:
If we want to release the GA build, please update this dependency. @jiancchen @palrahul